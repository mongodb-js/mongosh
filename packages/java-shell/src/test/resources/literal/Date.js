// command dontCheckValue
new Date()
// command dontCheckValue
Date()
// command checkResultClass
new Date(1)
// command checkResultClass
Date(1)
// command checkResultClass
new Date(1970, 2)
// command checkResultClass
new Date(1970, 2, 3, 4, 5)
// command checkResultClass
new Date("2012-12-19T06:01:17.171Z")
// command checkResultClass
new Date("2012-12-19")
// command checkResultClass
Date("2012-12-19")
// command checkResultClass
new Date({})