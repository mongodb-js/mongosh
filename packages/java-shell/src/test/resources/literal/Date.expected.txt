DateResult
StringResult
DateResult: { "$date" : 1}
StringResult: 1970-01-01T00:00:00.001Z
DateResult: { "$date" : 2678400000}
DateResult: { "$date" : 2865900000}
DateResult: { "$date" : 1355896877171}
DateResult: { "$date" : 1355875200000}
StringResult: 2012-12-19T00:00:00Z
org.graalvm.polyglot.PolyglotException: Expected number of string. Got: {} (Document{{}})